modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,93 @@ If you have installed the {gitops-title} Operator in the default namespace, run
+
+* Before this update, when the desired state for the server service type changes, for example, from `ClusterIP` to `NodePort`, the change was not applied automatically to the cluster without first deleting the resource. This update fixes the issue by adding a server service check to apply changes to the server service. If the desired state for the server service type changes, it is applied automatically to the cluster without deleting the resource. link:https://issues.redhat.com/browse/GITOPS-5385[GITOPS-5385]
+
+* Before this update, users were unable to set a custom image for the `ApplicationSet` Controller after specifying it in the Argo CD CR. This update fixes the issue by allowing users to set a custom image for the `ApplicationSet` Controller. link:https://issues.redhat.com/browse/GITOPS-5395[GITOPS-5395]
+
+* Before this update, Argo CD was unable to retrieve the correct TLS certificate for Helm OCI registries if the URL contained a path or port number. This update fixes the issue by using a fix in upstream Argo CD that ensures the URL is correctly parsed, allowing Argo CD to retrieve and return a valid TLS certificate. link:https://issues.redhat.com/browse/GITOPS-5081[GITOPS-5081]
+
 [id="breaking-change-1-15-0_{context}"]
 == Breaking change
 
 [id="removal-of-application-manager-cli-kam_{context}"]
 === Removal of {gitops-title} Application Manager CLI, kam
 
-* With this release, support for the {gitops-title} Application Manager command-line interface (CLI), `kam`, has been removed because this feature is no longer supported. As a result, all instances of `kam` have been removed from the {gitops-title} documentation. As an alternative to `kam`, you can use the Argo CD CLI that is available from {gitops-title} Operator v1.12.
\ No newline at end of file
+* With this release, support for the {gitops-title} Application Manager command-line interface (CLI), `kam`, has been removed because this feature is no longer supported. As a result, all instances of `kam` have been removed from the {gitops-title} documentation. As an alternative to `kam`, you can use the Argo CD CLI that is available from {gitops-title} Operator v1.12. link:https://issues.redhat.com/browse/GITOPS-4466[GITOPS-4466]
+
+[id="updated-openshift-api-schemas-and-insecureEdgeTerminationPolicy-field-change_{context}"]
+=== Updated OpenShift API schemas and `insecureEdgeTerminationPolicy` field change
+
+* With this update, the OpenShift API schemas used by the {gitops-title} Operator have been updated to include the latest fields from the Route API, resulting in a breaking change. If the `.spec.route.tls.insecureEdgeTerminationPolicy` field is configured with the *Disable* option in the ArgoCD Custom Resource (CR), it must be updated to *None* before upgrading to ensure compatibility with the new schema. link:https://issues.redhat.com/browse/GITOPS-5808[GITOPS-5808]
```

Comment:
```suggestion
* With this update, the OpenShift API schemas used by the {gitops-title} Operator have been updated to include the latest fields from the Route API, resulting in a breaking change. If the `.spec.*.route.tls.insecureEdgeTerminationPolicy` field is configured with the *Disable* option in the ArgoCD Custom Resource (CR), it must be updated to *None* before upgrading to ensure compatibility with the new schema. link:https://issues.redhat.com/browse/GITOPS-5808[GITOPS-5808]
```
Same comment as previous. Use wildcard `*` or list the fields individually?

Reply:
Using a wildcard causes issues in the build and results in build failure. I would like to list them individually.

Reply:
Fixed.